tests/integration/auth.spec.coffee

```diff
@@ -33,14 +33,7 @@ describe 'SDK authentication', ->
 			it 'should eventually be a valid api key given valid credentials', ->
 				resin.auth.authenticate(credentials)
 				.then(resin.auth.loginWithToken)
-				.then ->
-					return resin.request.send
-						method: 'POST'
-						url: '/api-key/user/full'
-						baseUrl: sdkOpts.apiUrl
-						body:
-							name: 'apiKey'
- 				.get('body')
+				.then(resin.auth.generateApiKey)
```

Comment:
Neat! Nice to see existing tests getting more useful and simpler at the same time :+1: